docs/manual/user/20_scanning.md

```diff
@@ -0,0 +1,192 @@
+# Scanning
+## Running a Scan with OpenSCAP
+
+#### Command Line Interface (CLI)
+This document outlines the usage of OpenSCAP, a command-line utility packaged within Fedora and Red Hat Enterprise Linux which allows users to load, scan, validate, edit, and export SCAP documents.
+
+See also [OpenSCAP User Manual](https://static.open-scap.org/openscap-1.3/oscap_user_manual.html) for instructions how to use OpenSCAP.
+Additional details regarding OpenSCAP can be found on the project homepage located at [open-scap.org](http://open-scap.org/).
+
+Five arguments to OpenSCAP are needed to perform a system scan against the upstream a profile:
+
+* `--profile`
+  * Mandatory, identifies which profile to scan against
+
+* `--results`
+  * Optional, indicates location to place ARF XML formatted results
+
+* `--report`
+  * Optional, indicates location to place HTML formatted results
+
+* datastream location
+  * Mandatory, identifies location of SCAP Source Datastream file
+
+Putting these arguments together, a properly formatted command would be:
+
+```
+$ sudo oscap xccdf eval --profile stig \
+--results /tmp/results.xml \
+--report /tmp/report.html \
+/usr/share/xml/scap/ssg/content/ssg-rhel7-ds.xml
+```
+
+While the scan is running, you will see output similar to the following on your screen:
+
+```
+Title   Install AIDE
+Rule    package_aide_installed
+Ident   CCE-27024-9
+Result  fail
+
+Title   Configure Periodic Execution of AIDE
+Rule    aide_periodic_cron_checking
+Ident   CCE-27222-9
+Result  notchecked
+
+Title   Verify File Permissions with RPM
+Rule    rpm_verify_permissions
+Ident   CCE-26731-0
+Result  fail
+
+Title   Verify File Hashes with RPM
+Rule    rpm_verify_hashes
+Ident   CCE-27223-7
+Result  pass
+```
+
+### Results Interpretation
+
+#### HTML Results
+
+Just open the `/tmp/report.html` file in your favorite browser.
+
+#### XML Results
+
+Looking at the `/tmp/results.xml` file, you will notice lines similar to those below:
+
+```xml
+    <rule-result idref="ensure_gpgcheck_globally_activated" time="2013-10-22T10:03:43" severity="high" weight="1.000000">
+      <result>pass</result>
+      <ident system="http://cce.mitre.org">CCE-26709-6</ident>
+      <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5">
+        <check-content-ref name="oval:ssg:def:413" href="ssg-rhel6-oval.xml"/>
+      </check>
+    </rule-result>
+    ......
+    <rule-result idref="package_aide_installed" time="2013-10-22T10:03:43" severity="medium" weight="1.000000">
+      <result>pass</result>
+      <ident system="http://cce.mitre.org">CCE-27024-9</ident>
+      <fix xmlns:xhtml="http://www.w3.org/1999/xhtml" system="urn:xccdf:fix:script:sh">
+        yum -y install aide
+      </fix>
+      <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5">
+        <check-content-ref name="oval:ssg:def:245" href="ssg-rhel6-oval.xml"/>
```

Comment:
```suggestion
        <check-content-ref name="oval:ssg:def:245" href="ssg-rhel7-oval.xml"/>
```

having rhel6 is ugly, but also rhel7 will eventually become deprecated as well :D

Reply:
I upgraded it to RHEL 9! :D